src/core/public/notifications/toasts/toasts_start_contract.tsx

```diff
@@ -68,18 +68,12 @@ export class ToastsStartContract {
   }
 
   public addWarning(toastOrTitle: ToastInput) {
-    return this.add({
-      color: 'warning',
-      iconType: 'help',
-      ...normalizeToast(toastOrTitle),
-    });
+    // Removed warning toasts per https://github.com/usePF/perchweb/issues/2031
+    return;
```

Comment:
While you were cruising through this repo did you see any place upstream from their notification system that generates the dangers/warnings? It would be good to get whatever Kibana considers "danger" into Sentry, I'm guessing that is what users are interpreting as errors (dark red box).

Reply:
I did not, but I can track that down. I was thinking it would make more sense to pass those errors to Sentry at the source rather than in the notification system since we'd have more valuable information at that point. By the time it gets here its just a single error line that isn't helpful at all.

Theoretically the way Sentry is configured it should be reporting those to Sentry already since the errors show in the browser console but I also don't see anything in Sentry for Perchybana.

Reply:
I'm wondering if Kibana is catching the errors and handling them which would prevent them from hitting Sentry. Either way we need to get Sentry hooked up and firin' away in Perchybana first so we can see what we're working with already, sounds like that might be happening today. I've seen a variety of "error" messages in Kibana, from what appears to be JS errors to things like "18 out of 20 nodes didn't respond" so I'm curious to what we will be getting with the out of box configuration.